### DISCUSSION TITLE
Allow the selection of one pointing device

### DISCUSSION THREAD

**Discussion by sqqqrly** [Softal-io/presentify-community#18] [Ideas]
I have multiple pointing devices on my Mac powerbook:

1. Touchpad on laptop
2. Magic trackpad
3. Wacom tablet (assigned to just one external monitor)

I wish I could configure Presentify to use only one of those devices. Usually that would be the Wacom.   This would be useful when presenting (ie Zoom) with shared screen.

**Comment by sqqqrly** [comment]
I am a software engineer and often present my work.   Presentify is useful with my Wacom, but its interface is a little clunky.  I have to use the keyboard to turn it on and off.   I want to be able to just pick up the Wacom pen and draw.  Then go back the magic pad and operate my computer normally.

**Comment by rampatra** [comment]
This looks like a smart request but I do not see a macOS API that gives me this information (i.e., current device that's being used to move your cursor around). Maybe macOS Accessibility API can provide this info with some hack but I do not want to request this permission in my app unless I really have to.

Regarding your other comment about clunkiness, can you pl elaborate a bit more and I will see if I can do something about it.

**Comment by sqqqrly** [comment]
Sorry. Clunky was a bit harsh.  I just meant that I have to go back and forth between devices (wacom, magic pad) and hit an F key to switch between Presentify and pointing device. 

This makes it harder than I like during customer demonstrations.    Practice helps muscle memory of course.  Presentify is still quite useful.

I will dig around for an API.

**Comment by sqqqrly** [comment]
Is this of help?
- https://stackoverflow.com/questions/854185/multiple-mice-on-os-x

**Comment by sqqqrly** [comment]
- https://github.com/icculus/manymouse

**Comment by neerajbadlani** [comment]
This is something I do quite often as well . And this will help . Thanks
